scripts/startTriggers.sh

```diff
@@ -23,9 +23,12 @@ SANDBOX_HOST="${1}"
 SANDBOX_PORT="${2}"
 DAR_FILE="${3:-/home/daml/bond-issuance.dar}"
 
+LIST_PARTIES_RESPONSE="$(daml ledger list-parties --host "${SANDBOX_HOST}" --port "${SANDBOX_PORT}" | tail -n +2)"
+PARTIES="$(echo "${LIST_PARTIES_RESPONSE}" | cut -d"{" -f2 | cut -d" " -f3 | cut -d"'" -f2)"
```

Comment:
let's simplify a bit, two ways:
1. "pure" bash
```suggestion
LIST_PARTIES_RESPONSE=$(daml ledger list-parties --host "$SANDBOX_HOST" --port "$SANDBOX_PORT")
PARTIES=$(echo "$LIST_PARTIES_RESPONSE" | grep -o "party = '.*'" | sed "s/party = '\(.*\)'/\1/")
```
2. or with `jq` parser:
```suggestion
LIST_PARTIES_RESPONSE=$(daml ledger list-parties --host "$SANDBOX_HOST" --port "$SANDBOX_PORT")
PARTIES=$(echo "$LIST_PARTIES_RESPONSE" | jq ".[].party" | sed s/\"//g)
```